lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -369,14 +369,13 @@ def add_creditcard(post, creditcard, options)
             card[:cvc] = creditcard.verification_value if creditcard.verification_value?
             card[:name] = creditcard.name if creditcard.name
           end
-          post[:card] = card
 
-          if creditcard.is_a?(NetworkTokenizationCreditCard) && creditcard.source == :apple_pay
-            post[:three_d_secure] = {
-              apple_pay:  true,
```

Comment:
Stripe doesn't want an indication that it's an apple_pay vs. (android pay?) card anymore?